nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestProcessorLifecycle.java

```diff
@@ -191,6 +193,7 @@ public void validateStopCallsAreMeaninglessIfProcessorNotStarted() throws Except
      * operations can only be @OnScheduled, @OnUnscheduled, @OnStopped.
      */
     @Test
+    @Category(IntegrationTest.class)
```

Comment:
This test (and the one following) were not marked as @Ignore before and aren't in a named integration test file (starting or ending in IT). Are these actually integration tests that have been running as unit tests, or do they just take too long to run, or are they issuing failures? In the last case, I recommend against labelling failing tests with the Category and keep the @Ignore so we can easily find them in source without having to run all (integration) tests to see which pass and which fail.